backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/webservice/dataloader/AcceptingStoreDataLoader.kt

```diff
@@ -14,14 +14,10 @@ val acceptingStoreLoader = DataLoader<Int, AcceptingStore?> { ids ->
         runBlocking {
             transaction {
                 AcceptingStoresRepository.findByIds(ids).map {
-                    AcceptingStore(it.id.value, it.name, it.description, it.contactId.value, it.categoryId.value)
-                }.associateWithKeys<AcceptingStore, Int>({ it.id }, ids)
+                    if (it == null) null
```

Comment:
This is a little bit weird. The accepting store shouldn't be null I think

Reply:
The user might always query ids for which the accepting store is null